runtime/doc/options.txt

```diff
@@ -5536,6 +5536,12 @@ A jump table for the options with a short description can be found at |Q_op|.
 	   "auto"   	only when there is a sign to display
 	   "auto:[1-9]" resize to accommodate multiple signs up to the
```

Comment:
you should remove this line then

Reply:
okay, should I update a new one to include this as well?

Reply:
nevermind let it like this